scripts/python/junglecatsoftware/awstoolkit/outputFormatters.py

```diff
@@ -24,3 +25,8 @@ def printStartDateStamp():
 def printIndented( text, indentLevel = 1 ):
   for line in text.splitlines():
     print( ( '\t' * indentLevel ) + line )
+
+if __name__ == "__main__":
```

Comment:
It's a python idiom.  if __name__ == "__main__": block only triggers if the module is executed directly.  (In which case __name__ will be "__main__").  If you import the module, the __name__ variable is the module name.  This is handy for triggering tests, and making other nice things happen.  I can tie in some "real" testing on this if you'd like.